File: LeetCode/P461Easy_Hamming_Distance/main.py
# Problem: https://leetcode.com/problems/hamming-distance/


class Solution:
    def hammingDistance(self, x: int, y: int) -> int:
        dist = 0
        while x or y:
            dist += (x & 1) != (y & 1)
            x >>= 1
            y >>= 1
        return dist

File: LeetCode/P461Easy_Hamming_Distance/main.c
// Problem: https://leetcode.com/problems/hamming-distance/

int hammingDistance(int x, int y) {
  int dist = 0;
  while (x || y) {
    dist += (x & 1) != (y & 1);
    x >>= 1;
    y >>= 1;
  }
  return dist;
}